catalog/imaging/imaging-plugin-nitf/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -20,11 +20,14 @@
            http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
            http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
 
+    <reference id="catalogFramework" interface="ddf.catalog.CatalogFramework"
+               availability="mandatory"/>
 
-    <bean id="plugin" class="org.codice.alliance.plugin.nitf.NitfPreStoragePlugin">
+    <bean id="plugin" class="org.codice.alliance.plugin.nitf.NitfPostIngestPlugin">
         <cm:managed-properties persistent-id="NITF_Render_Plugin"
                                update-strategy="container-managed"/>
         <property name="maxSideLength" value="1024"/>
```

Comment:
I don't see the `maxSideLength` getter/setter methods in the plugin anymore. Should this property be removed?

Reply:
I think i might have accidently removed the setter because intellij was telling me nothing used it anymore. Will add it back in.